tern/analyze/common.py

```diff
@@ -32,11 +32,18 @@
 
 def get_shell_commands(shell_command_line):
     '''Given a shell command line, get a list of Command objects'''
-    comm_list = general.split_command(shell_command_line)
-    cleaned_list = []
-    for comm in comm_list:
-        cleaned_list.append(Command(general.clean_command(comm)))
-    return cleaned_list
+    statements = general.split_command(shell_command_line)
```

Comment:
Here we use the shell script parser to get statements, and then pick out `commands` and `commands in a loop`. From each `command` statement, we append it to the `command_list`.